raiden-cli/src/routes/tokens.ts

```diff
@@ -1,20 +1,52 @@
 import { Router, Request, Response } from 'express';
+import { first } from 'rxjs/operators';
+
 import { Cli } from '../types';
+import { validateAddressParameter } from '../utils/validation';
+
+async function getAllTokens(this: Cli, _request: Request, response: Response) {
+  response.json(await this.raiden.getTokenList());
+}
+
+async function getTokenNetwork(this: Cli, request: Request, response: Response) {
+  // Attention: this also enables monitoring token
+  try {
+    response.json(await this.raiden.monitorToken(request.params.tokenAddress));
```

Comment:
Hmm. I'm not sure if this is an expected side effect. I would maybe just add it when it becomes necessary for the scenario player or so. Until then I would stick with the spec.

Reply:
I am sticking to the spec. The spec has no notion of 'monitored' tokens or not, Raiden python always monitors all tokens. We just don't before as an optimization, but the only method to extract the `tokenNetwork` address is by also monitoring it, and I think it's sensible, monitoring a token don't take us farther from the python behavior, but instead closer.

Reply:
Hmm okay. :shrug: 
Thanks that you answered with a good argumentation. :pray: